### PR TITLE
Adding myExperiment logo on login screen

### PR DESCRIPTION
**Pull request by Hiteshgautam01**

### CODE REVIEW

**Comment by Hiteshgautam01** [comment]
Login screen UI after improvements

tavernascreen1

**Comment by sagar15795** [comment]
@stain @ianwdunlop Can we use the myexperiment logo in the app?

**Comment by ianwdunlop** [comment]
I think it is ok to use the logo as long as we have some disclaimer pointing out that myExperiment is nothing to do with Apache and is copyright (whoever it is, maybe Manchester University). Maybe an infobox popup next to the logo would help do this. Plus add it to the notice page.

**Comment by Hiteshgautam01** [comment]
Okay, @ianwdunlop I will add an infobox popup next to the logo and add to the notice page.
@sagar15795 what we have to write in that infobox and notice page.
 and @ianwdunlop @stain, I have shared my proposal draft. So, can you please review it.

**Comment by sagar15795** [comment]
@ianwdunlop we need to write the disclaimer in the app or in the project readme

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795  i have added the info button on the right side of the logo. UI after this addition.

tavernascreen1 copy

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -60,16 +60,13 @@
...
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
+                android:layout_height="@dimen/myExperimentLogoHeight"
                 android:layout_gravity="center"
-                android:background="@color/grey_background"
-                android:padding="@dimen/login_info_padding"
-                android:text="@string/logininfo"
-                android:textSize="@dimen/login_text_size"
-                android:textStyle="normal|bold" />
+                android:src="@drawable/myexlogo"
```
Comment (saketkumar95) [inline app/src/main/res/layout/fragment_login.xml]: @Hiteshgautam01 I wonder that this image might be small on large devices like android tablet. Can you please do a quick check that this looks good at the too large or too small devices?

**Comment by Hiteshgautam01** [comment]
@sagar15795 Please review

**Comment by sagar15795** [comment]
Add this to bottom of the login screen in small font. 
`Disclaimer myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment`

Also, add it to Notice section.

**Comment by Hiteshgautam01** [comment]
Hey, @sagar15795  This is the UI after changes and additions. please review then I will commit the changes.

screen1234taverna

**Comment by sagar15795** [comment]
@Hiteshgautam01 This looks good. Also, add it to notice section.

**Comment by Hiteshgautam01** [comment]
Hey, @sagar15795 I have done the requested changes. Please review.

File: app/src/main/assets/apache_licence_notice.html [hunk 2]
```diff
@@ -229,6 +229,8 @@
     Apache Taverna Mobile
 Copyright 2015-2018 The Apache Software Foundation
 
+1.Disclaimer: myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment.
```
Comment (sagar15795) [inline app/src/main/assets/apache_licence_notice.html]: move it line no 236. just add `myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment.`.

Comment (Hiteshgautam01) [reply app/src/main/assets/apache_licence_notice.html]: I have done the changes.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -232,6 +232,10 @@ limitations under the License.
     <string name="pref_key_myexperiment_pass">pref_myexperimentpass</string>
     <string name="pref_summary_myexperiment_pass">Myexperiment password</string>
 
+    <!--Disclaimer:-->
+    <string name="DisclaimerTitle">Disclaimer:</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: @Hiteshgautam01 Please follow this coding guidelines, Here your strings resource name should be 
`disclaimer_title` not `DisclaimerTitle`. Please make the change.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -232,6 +232,10 @@ limitations under the License.
...
+    <string name="DisclaimerContent">myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: same as above strings resource name should be `disclaimer_content` not `DisclaimerContent`.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -32,7 +32,7 @@ limitations under the License.
         <item>Date Last Run</item>
     </array>
     <string name="hello_blank_fragment">About Workflow</string>
-    <string name="logininfo">myExperiment Login</string>
+    <string name="logininfo">Login</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: make the `logininfo` to `login_info`.

File: app/src/main/res/values/dimens.xml [hunk 6]
```diff
@@ -71,15 +71,18 @@ limitations under the License.
...
+    <dimen name="myExperimentLogoHeight">50dp</dimen>
     <dimen name="login_text_size">20sp</dimen>
     <dimen name="input_layout_email_margin">20dp</dimen>
     <dimen name="loginButton_marginBottom">8dp</dimen>
     <dimen name="loginButton_marginTop">10dp</dimen>
     <dimen name="appName_text_margin">12dp</dimen>
     <dimen name="title_text_size">40sp</dimen>
     <dimen name="logo_margin_top">40dp</dimen>
+    <dimen name="discTextsize">5dp</dimen>
+    <dimen name="discMarginTop">10dp</dimen>
```
Comment (therajanmaurya) [inline app/src/main/res/values/dimens.xml]: please rename the resource name with `disc_margin_top`

File: app/src/main/res/values/dimens.xml [hunk 7]
```diff
@@ -71,15 +71,18 @@ limitations under the License.
...
+    <dimen name="login_info_padding">7dp</dimen>
+    <dimen name="myExperimentLogoHeight">50dp</dimen>
     <dimen name="login_text_size">20sp</dimen>
     <dimen name="input_layout_email_margin">20dp</dimen>
     <dimen name="loginButton_marginBottom">8dp</dimen>
     <dimen name="loginButton_marginTop">10dp</dimen>
     <dimen name="appName_text_margin">12dp</dimen>
     <dimen name="title_text_size">40sp</dimen>
     <dimen name="logo_margin_top">40dp</dimen>
+    <dimen name="discTextsize">5dp</dimen>
```
Comment (therajanmaurya) [inline app/src/main/res/values/dimens.xml]: please rename the resource name with `disc_text_size `

File: app/src/main/res/values/dimens.xml [hunk 8]
```diff
@@ -71,15 +71,18 @@ limitations under the License.
     <dimen name="layout_padding">8dp</dimen>
     <dimen name="text_size">25sp</dimen>
     <dimen name="login_layout_padding">20dp</dimen>
-    <dimen name="login_layout_marginTop">20dp</dimen>
-    <dimen name="login_info_padding">10dp</dimen>
+    <dimen name="login_layout_marginTop">15dp</dimen>
+    <dimen name="login_info_padding">7dp</dimen>
+    <dimen name="myExperimentLogoHeight">50dp</dimen>
```
Comment (therajanmaurya) [inline app/src/main/res/values/dimens.xml]: please rename the resource name with `my_experiment_logo_height`

**Comment by Hiteshgautam01** [comment]
Hey, @therajanmaurya  @sagar15795 I have done all the requested changes. Please review.

**Comment by afs** [comment]
It should say "Apache Taverna".

**Comment by Hiteshgautam01** [comment]
Hey @afs, thanks for notifying such a small mistake. I have done the changes.

**Pull request merged**